open_seq2seq/encoders/w2l_encoder.py

```diff
@@ -0,0 +1,157 @@
+# Copyright (c) 2018 NVIDIA Corporation
+from __future__ import absolute_import, division, print_function
+from __future__ import unicode_literals
+from six.moves import range
+
+import tensorflow as tf
+
+from .encoder import Encoder
+from open_seq2seq.parts.cnns.conv_blocks import conv_bn_actv 
+
+class Wave2LetterEncoder(Encoder):
+  """Wave2Letter like encoder."""
```

Comment:
We usually have the same """  ...   """ block for entire documentation string. Like this should be changed to
"""Wave2Letter like encoder. Fully convolutional model.""" or you can keep the newline if you want: 
"""Wave2Letter like encoder. 

Fully convolutional model.
"""